crates/astria-composer/tests/blackbox/composer.rs

```diff
@@ -61,22 +60,33 @@ async fn tx_from_two_rollups_are_received_by_sequencer() {
 /// sequence action is for the given `expected_chain_id`.
 async fn mount_broadcast_tx_sync_mock(
     server: &MockServer,
-    expected_chain_id: &'static str,
+    expected_chain_name: &'static str,
```

Comment:
Changed this to better disambiguate between the chain name and its ID which was derived from the name.